charts/aws-ebs-csi-driver/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Helm chart
 
+## v2.7.1
```

Comment:
Seems like you forgot to rebase? :)

Reply:
Oops, did this before v1.8.0 was merged in. Fixed!